desk/app/groups.hoon

```diff
@@ -3463,7 +3451,7 @@
         ?.  ?=(kind:d p.nest)  |
         .^(? %gu (weld pre /v3/[p.nest]/(scot %p p.q.nest)/[q.q.nest]))
       =?  active-channels.group  active
-        (~(put by active-channels.group) nest)
+        (~(put in active-channels.group) nest)
```

Comment:
Classic. I wonder if we can come up with a way to catch this stuff automatically/earlier.